### PR TITLE
'integration freeze' must print datadog packages only

### PR DESCRIPTION
**Pull request by FlorianVeaux**
### What does this PR do?

This PR changes the behavior of the 'integration freeze' cli command. Instead of displaying the result of `pip freeze` with all the python packages, the command will only shows datadog packages.

### Motivation
Since this is a `agent integration` command, it should only show datadog packages and nothing else from the embedded python.

### Additional Notes
--

### CODE REVIEW

**Review by trishankatdatadog — DISMISSED** [review]
LGTM

**Comment by FlorianVeaux** [comment]
^ Fixing a small typo in the changelog

**Review by nmuesch — DISMISSED** [review]
This looks good to me. Lets get a 👍 from @DataDog/agent-core too

File: cmd/agent/app/integrations.go [hunk 1]
```diff
@@ -238,26 +240,26 @@ func pip(args []string) error {
 	pipCmd := exec.Command(pythonPath, args...)
 
 	// forward the standard output to stdout
-	stdout, err := pipCmd.StdoutPipe()
+	pip_stdout, err := pipCmd.StdoutPipe()
```
Comment (hush-hush) [inline cmd/agent/app/integrations.go]: Go convention is to use mixedCaps https://golang.org/doc/effective_go.html#mixed-caps for variables name (surprised `go vet` didn't raised an issue).

Could you update the PR to follow the convention ?

Comment (FlorianVeaux) [reply cmd/agent/app/integrations.go]: Sure thing 👍

**Review by hush-hush — APPROVED** [review]

**Pull request merged**